models/intel/person-detection-0203/description/person-detection-0203.md

```diff
@@ -0,0 +1,52 @@
+# person-detection-0203
+
+## Use Case and High-Level Description
+
+This is a person detector that is based on MobileNetV2
+backbone with ATSS head for 864x480 resolution.
+
+## Example
+
+![](./person-detection-0203.png)
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| AP @ [ IoU=0.50:0.95 ]          | 0.412 (internal test set)                 |
+| GFlops                          | 6.519                                     |
+| MParams                         | 2.394                                     |
+| Source framework                | PyTorch\*                                 |
+
+Average Precision (AP) is defined as an area under
+the [precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve.
+
+## Performance
```

Comment:
we are removing empty sections from documentation (see [PR-1749](https://github.com/openvinotoolkit/open_model_zoo/pull/1749)), please do that as well